admin/app/assets/javascripts/workarea/admin/modules/select2.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
im confused by the addition of this file, but I'll leave the judging to Curt

Reply:
The idea is to allow you to make a pure instance of select2 for things like inline options like `allowClear`, etc. `remoteSelects` is written specifically for remote stuff: https://github.com/workarea-commerce/workarea/blob/master/admin/app/assets/javascripts/workarea/admin/modules/remote_selects.js

Reply:
@mttdffy this is something we had originally, long, long ago, when Select2 was introduced into the Admin, but we had no real use case for it (until now) so it fell by the wayside. As part of #142 I'll better align remoteSelects & select2's APIs so this makes a bit more sense.